### ISSUE TITLE
Remove event as default feature, until finished

### ISSUE THREAD

**Issue by emilvincentz** [acterglobal/a3#937]
As event is not yet working, i think we should disable it as a default feature until we have 
1. made the rsvp work without the app crashing 
2. can invite people to an event. 
3. We only show the events that a user has rsvp'd to in the home screen 
4. make events and the event calendar show up below the spaces on the users home screen (right now it takes up so much space that you cant see your spaces + the calendar makes it difficult to scroll down)

Below video demostrating that 1. your spaces are hidden 2. difficult to scroll due to calendar

https://user-images.githubusercontent.com/77483554/267427501-7c975fe3-fb8d-461d-ab92-73914b35f231.mov

**Comment by gnunicorn** [comment]
> can invite people to an event.

what does that mean? Any user can see all events from all spaces they are in - they are _invited_ ...

**Comment by emilvincentz** [comment]
@gnunicorn  <img width="785" alt="Screenshot 2023-09-13 at 10 22 47" src="https://github.com/acterglobal/a3/assets/77483554/f56f1a45-d0a1-4475-9d4a-147c7a1fe10b">

**Comment by gnunicorn** [comment]
Well, I wasn't asking what you mean by "it doesn't work", but what "do you mean by inviting". I don't know what is supposed to happen when you clicked that button. I am fine removing it. That would remove the error-message ...

**Comment by emilvincentz** [comment]
Ah I understand now. 
In that case, the invite button is for inviting people that is not already part of the spaces, but e.g. member of the parent space. However I dont think thats a top priorate feature as of now. Most importantly that people can rsvp and that not all events that you are invited to across spaces show up in the home screen. 
So In that case, I guess we can go ahead and remove the "invite" and "join" button, correct?

**Comment by gnunicorn** [comment]
> but e.g. member of the parent space

Yeah, as mentioned everything that goes to "outside the current space" is not yet clear how to do that technically. There is a lot to consider, also regarding privacy and encryption, before we can do much like that ...

> Most importantly that people can rsvp and that not all events that you are invited to across spaces show up in the home screen.

Is there anything not yet covered by #800 ? Let's add it there...

**Comment by emilvincentz** [comment]
No I think it pretty well covered as far as I can see

**Comment by gnunicorn** [comment]
closing in favor of #800
